e2e/next/src/next.test.ts

```diff
@@ -177,6 +177,12 @@ describe('Next.js Applications', () => {
       `dist/packages/${appName}/public/shared/ui/hello.txt`
     );
 
+    // Check that compiled next config does not contain bad imports
+    const nextConfigPath = `dist/packages/${appName}/next.config.js`;
```

Comment:
I thought we were adding a `lint` rule. Is this just a stand-in until we have that set up?

Reply:
I'll add the lint rule, but also want to make sure it doesn't get into the built file in case we missed something in lint.